src/app/game-state/main-loop.service.ts

```diff
@@ -7,6 +7,7 @@ import { OfflineModalComponent } from '../offline-modal/offline-modal.component'
 
 const TICK_INTERVAL_MS = 25;
 const LONG_TICK_INTERVAL_MS = 500;
+const BACKGROUND_TICK_INTERVAL_MS = 1 * 1000;
```

Comment:
Why not just 1000?

Reply:
I was experimenting with longer times and this was an artifact of that experimentation, but 1s seems to work and longer times more often result in lag when you bring the tab up again.